.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Go
+on: [push]
+jobs:
+  run:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    env:
+      OS: ${{ matrix.os }}
```

Comment:
is the envvar export of `OS` used by `go` or `codecov`?

Reply:
good point. Removed.